### PR TITLE
Koin from swift

### PR DESCRIPTION
**Pull request by nis-ship-it**
- #42

- Well can now call the koin dependencies that are registered from swift code!!

### CODE REVIEW

File: bisqapps/iosClient/Podfile.lock [hunk 1]
```diff
@@ -18,4 +18,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: e4e5bb187d8247572973af90a35b57b84c96b4c6
 
-COCOAPODS: 1.16.2
+COCOAPODS: 1.15.2
```
Comment (rodvar) [inline bisqapps/iosClient/Podfile.lock:21]: is this intentional? if so, happy to merge it but just would like to understand why the downgrade?

File: bisqapps/shared/presentation/src/iosMain/kotlin/network/bisq/mobile/presentation/di/PresentationDIHelper.kt [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
 import network.bisq.mobile.domain.di.domainModule
+import org.koin.core.Koin
 import org.koin.core.context.startKoin
+import org.koin.core.parameter.parametersOf
+import org.koin.core.qualifier.Qualifier
 
-fun initKoin() {
-    startKoin {
-        modules(listOf(domainModule, presentationModule))
+class DependenciesProviderHelper {
```
Comment (rodvar) [inline bisqapps/shared/presentation/src/iosMain/kotlin/network/bisq/mobile/presentation/di/PresentationDIHelper.kt:10]: let's rename the file to match the new class name please

File: bisqapps/kls_database.db
Comment (rodvar) [inline bisqapps/kls_database.db:1]: do we need this file? if so, why? otherwise please remove

**Review by rodvar — CHANGES_REQUESTED** [review]
@nis-ship-it well done nish!

Just some needs for your consideration please see my comments

File: bisqapps/iosClient/Podfile.lock (hunk 1 above)
Comment (nis-ship-it) [reply bisqapps/iosClient/Podfile.lock:21]: Nope!! automated

Comment (nis-ship-it) [reply bisqapps/iosClient/Podfile.lock:21]: i didnt have up to date cocoapods

File: bisqapps/kls_database.db
Comment (nis-ship-it) [reply bisqapps/kls_database.db:1]: nope, removed it

File: bisqapps/iosClient/Podfile.lock (hunk 1 above)
Comment (rodvar) [reply bisqapps/iosClient/Podfile.lock:21]: ok I'll revert that then, let me just test the change without this

**Review by rodvar — APPROVED** [review]

**Comment by rodvar** [comment]
@nis-ship-it cool, I've reverted the downgrade on the pods, just waiting for the CI to pass to merge 💪

**Pull request merged**